backend/server.py

```
@@ -6634,6 +6634,12 @@ async def get_doctor_diagnostics(
 from agents.portfolio_api import portfolio_router
 app.include_router(portfolio_router)
 
+try:
+    from agents.agile_api import agile_router
+    app.include_router(agile_router)
+except Exception as _agile_err:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)